src/Microsoft.AspNetCore.Hosting.Abstractions/IWebHostBuilder.cs

```diff
@@ -39,6 +40,13 @@ public interface IWebHostBuilder
         IWebHostBuilder ConfigureLogging(Action<ILoggerFactory> configureLogging);
 
         /// <summary>
+        /// Adds a delegate for configuring the provided <see cref="ILoggerFactory"/>. This may be called multiple times.
+        /// </summary>
+        /// <param name="configureLogging">The delegate that configures the <see cref="ILoggerFactory"/>.</param>
+        /// <returns>The <see cref="IWebHostBuilder"/>.</returns>
+        IWebHostBuilder ConfigureLogging<T>(Action<T> configureLogging) where T : ILoggerFactory;
```

Comment:
This *does* bug me a bunch. The fact that your delegate may never run because T wasn't what you expected but maybe that's fine. We never figured out what to do here right?